src/main/java/com/google/cloud/tools/appengine/api/exceptions/NullSdkPathException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.api.exceptions;
+
+/**
+ * Signals that a given SDK path is null.
+ */
+public class NullSdkPathException extends AppEngineException {
```

Comment:
I thought about this but, then, we wouldn't be throwing an AppEngineException anymore. Would you still rather have an NPE?